### ISSUE TITLE
The game crashes at 90 turn

### ISSUE THREAD

**Issue by Fasysta** [yairm210/Unciv#8068]
**Platform**
Windows, 64 bit

**Version**
4.3.4

**Describe the bug**
The game crashes at 98 turn

**Expected behavior**
Help me

**Platform:** Desktop
**Version:** 4.3.4 (Build 769)
**Rulesets:** [Civ V - Vanilla, Civ V - Gods & Kings, Community Maps]
**Last Screen:** `com.unciv.ui.worldscreen.WorldScreen`

--------------------------------

OS: Windows 10
Java: Oracle Corporation 25.232-b09
	Max Memory: 910 MB

--------------------------------

**Message:**
```
java.lang.NullPointerException
	at com.unciv.ui.worldscreen.unit.UnitActions.getRepairTurns(UnitActions.kt:506)
	at com.unciv.ui.worldscreen.unit.UnitActions.access$getRepairTurns(UnitActions.kt:35)
	at com.unciv.ui.worldscreen.unit.UnitActions$getRepairAction$1.invoke(UnitActions.kt:536)
	at com.unciv.ui.worldscreen.unit.UnitActions$getRepairAction$1.invoke(UnitActions.kt:533)
	at com.unciv.logic.automation.unit.SpecificUnitAutomation.automateImprovementPlacer(SpecificUnitAutomation.kt:286)
	at com.unciv.logic.automation.unit.UnitAutomation.automateUnitMoves(UnitAutomation.kt:189)
	at com.unciv.logic.automation.civilization.NextTurnAutomation.automateUnits(NextTurnAutomation.kt:885)
	at com.unciv.logic.automation.civilization.NextTurnAutomation.automateCivMoves(NextTurnAutomation.kt:84)
	at com.unciv.logic.GameInfo.nextTurn(GameInfo.kt:275)
	at com.unciv.ui.worldscreen.WorldScreen$nextTurn$1.invokeSuspend(WorldScreen.kt:592)
	at com.unciv.ui.worldscreen.WorldScreen$nextTurn$1.invoke(WorldScreen.kt)
	at com.unciv.ui.worldscreen.WorldScreen$nextTurn$1.invoke(WorldScreen.kt)
	at com.unciv.utils.concurrency.ConcurrencyKt$launchCrashHandling$1.invokeSuspend(Concurrency.kt:83)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at com.unciv.utils.concurrency.CrashHandlingDispatcher$dispatch$1.invoke(Concurrency.kt:164)
	at com.unciv.utils.concurrency.CrashHandlingDispatcher$dispatch$1.invoke(Concurrency.kt:164)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandling$1.invoke(CrashHandlingExtensions.kt:17)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandlingUnit$1.invoke(CrashHandlingExtensions.kt:33)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandlingUnit$1.invoke(CrashHandlingExtensions.kt:33)
	at com.unciv.utils.concurrency.CrashHandlingDispatcher.dispatch$lambda-0(Concurrency.kt:164)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

```
**Save Mods:**
```
[Civ V - Gods & Kings]
```

**Save Data:**
<details><summary>Show Saved Game</summary>

**Comment by Ouaz** [comment]
This has already been fixed (that was fast ^^): https://github.com/yairm210/Unciv/pull/8065

You can download 4.3.4-patch1 just released.

**Comment by itanasi** [comment]
> This has already been fixed (that was fast ^^): #8065
Kinda watching bugs like a hawk aince I made so many edits haha

**Comment by Fasysta** [comment]
> This has already been fixed (that was fast ^^): #8065
> 
> You can download 4.3.4-patch1 which has the fix.

thank you!
